Add 2025 Community Update (#2416)
writethedocs/www: docs/blog/2025-community-update.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: Aug 8, 2025
   :tags: stats, newsletter
   :author: Eric Holscher

Write the Docs Community Update - August 2025
=============================================

Hey there documentarians!

Eric Holscher here, sharing this year's community update as we reflect on another year of growth and change for Write the Docs.

> We've been heavily focused on conferences this year, but our Slack community, Newsletter, Meetups, and other initiatives continue without too much change. 

Reviewer: ```suggestion
We've been heavily focused on conferences this year, but our Slack community, Newsletter, Meetups, salary survey, and other initiatives continue without major changes. 
```

Author: This isn't a full team, so feels odd to mention here? I think it's covered in other initiatives? I mostly just wanted to cover each of the teams on the team page to make sure we included them in the note.

> Portland was a great success
> ----------------------------

Reviewer: ```suggestion
WTD Portland was a great success
-------------------------------
```

Author: This feels repetitive? I think people realize we're talking about ourselves?

Reviewer: For sure, but it just reads odd as if it's the city. Could use something like 

> The Portland conference was a great success 

instead?

Author: Yea, the mix between Geos, Countries, and Cities is always weird. Branding is hard.

Our Portland conference was our largest yet, with 415 in-person attendees and 170 virtual participants.
New features like Sunday registration location, food carts, and expanded venue spaces helped make the experience even better.
All talk videos are available on our `YouTube playlist <https://www.youtube.com/playlist?list=PLZAeFn6dfHplMbtJtidqFFtL7rt3ASNSR>`_, and you can view the photos in our `Flickr album <https://www.flickr.com/photos/writethedocs/albums/72177720325861452/>`_.
We're excited to see everyone again in Portland in 2026,
and will be announcing the dates soon.

> Berlin returns in-person
> ------------------------

Reviewer: ```suggestion
WTD Europe is back, in Berlin
-----------------------------------
```

> The biggest news for 2025 is the return of our European conference to Berlin!  

Reviewer: ```suggestion
The biggest news for 2025 is the return of our European conference, now in Berlin!  
```

After several years of virtual-only events, we're thrilled to be back in person in Europe for the first time since 2019.  
The Berlin conference will also feature a virtual component, so documentarians can join us from anywhere.  
Find out more and get your tickets on the `Berlin website <https://www.writethedocs.org/conf/berlin/2025/>`_.  
Thanks to our local team and everyone who helped make this possible!

> Africa continues to grow
> ------------------------

Reviewer: ```suggestion
WTD Africa continues to grow
-----------------------------
```

> We're excited to see our Africa community continue to grow, with Kenya hosting its second event and first conference and more local events popping up.  

Reviewer: ```suggestion
We're excited to see our Africa community continue to grow, with Kenya hosting its second event and first conference, and more local events popping up.  
```
